CHANGELOG

```diff
@@ -137,6 +137,8 @@ Linear and nonlinear solvers
 - Added partially assembled convergent diagonal preconditioner for adaptively
   refined meshes (i.e. non-conforming finite element spaces), see Example 6/6p.
 
+- Upgraded SuperLU interface to use SuperLU_DIST 6.3.1
```

Comment:
The version number is not consistent throughout, is it 6.3.1 or 6.3.0 ?

Reply:
I'll make it consistent.  It should work with both, and the SuperLU folks should be dropping 6.3.1 soon.